### ISSUE TITLE
Social Media Widget Icons WAI-ARIA Role of "img" Fails Accessibility Checks

### ISSUE THREAD

**Issue by kellyhelming** [Automattic/jetpack#12338]
#### Is your feature request related to a problem? Please describe.
Please amend the WAI-ARIA role to comply with WCAG requirements: https://www.w3.org/TR/WCAG20-TECHS/ARIA6.html

1.1.1 Non-text Content
Description of this issue:
The WAI-ARIA image, created with role="img", does not have an accessible name.

#### Describe the solution you'd like
How to fix it:
If the image is informative or functional, use either the WAI-ARIA attribute 'aria-label' or 'aria-labelledby' to create an alternative text for the image.

If the image is purely decorative, use the WAI-ARIA attribute role="presentation" instead of role="img" for the image. This will cause assistive technologies to ignore the image.

#### Describe alternatives you've considered
This solution was tested with via Chrome's "inspect" with Siteimprove's checker plug-in. When the role was changed from "img" to "presentation", the errors cleared.

#### Additional context

![Screen Shot 2019-05-10 at 9 21 32 AM](https://user-images.githubusercontent.com/17484411/57542191-d1004f80-7305-11e9-8e0d-4b5f43b1f55b.png)
